pkg/koordlet/statesinformer/states_informer.go

```diff
@@ -170,6 +178,13 @@ func (s *statesInformer) Run(stopCh <-chan struct{}) error {
 	go s.startCallbackRunners(stopCh)
 
 	go wait.Until(s.reportNodeTopology, s.config.NodeTopologySyncInterval, stopCh)
+	if features.DefaultKoordletFeatureGate.Enabled(features.Accelerators) {
+		go wait.Until(s.reportDevice, s.config.NodeTopologySyncInterval, stopCh)
+		// check is nvml is available
+		if s.initGPU() {
```

Comment:
If init fails, do you still need to start the reportDevice goroutine?

Reply:
node may contains fpga, but not contains gpu